gunicorn/tests/conftest.py

```diff
@@ -0,0 +1,93 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+import subprocess
+import pytest
+import os
+import tempfile
+import requests
+import shutil
+import logging
+import time
+
+from common import FIXTURES, PROC_NAME
+
+log = logging.getLogger('test_gunicorn')
+
+
+@pytest.fixture(scope="session")
+def setup_gunicorn(request):
+    gunicorn_tmpdir = tempfile.mkdtemp()
```

Comment:
You can do https://github.com/DataDog/integrations-core/blob/master/openldap/tests/conftest.py#L22